backend/MoleculeOSSite/Properties/launchSettings.json

```diff
@@ -1,31 +1,45 @@
-﻿{
-    "$schema": "https://json.schemastore.org/launchsettings.json",
-    "iisSettings": {
-        "windowsAuthentication": false,
-        "anonymousAuthentication": true,
-        "iisExpress": {
-            "applicationUrl": "http://localhost:26063",
-            "sslPort": 44333
-        }
+{
+  "$schema": "https://json.schemastore.org/launchsettings.json",
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:26063",
+      "sslPort": 44333
+    }
+  },
+  "profiles": {
+    "MoleculeOSSite": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "launchUrl": "swagger",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "applicationUrl": "https://localhost:7108;http://localhost:5200",
+      "dotnetRunMessages": true
+    },
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "launchUrl": "swagger",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "WSL 2": {
+      "commandName": "WSL2",
+      "launchBrowser": true,
+      "launchUrl": "https://localhost:7108/swagger",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "ASPNETCORE_URLS": "https://localhost:7108;http://localhost:5200"
+      },
+      "distributionName": ""
     },
-    "profiles": {
-        "MoleculeOSSite": {
-            "commandName": "Project",
-            "dotnetRunMessages": true,
-            "launchBrowser": true,
-            "launchUrl": "swagger",
-            "applicationUrl": "https://localhost:7108;http://localhost:5200",
-            "environmentVariables": {
-                "ASPNETCORE_ENVIRONMENT": "Development"
-            }
-        },
-        "IIS Express": {
-            "commandName": "IISExpress",
-            "launchBrowser": true,
-            "launchUrl": "swagger",
-            "environmentVariables": {
-                "ASPNETCORE_ENVIRONMENT": "Development"
-            }
-        }
+    "Docker": {
+      "commandName": "Executable",
+      "executablePath": "C:\\Programowanie\\C#\\Molecule\\moleculeos-site\\development.sh"

```

Comment:
Ustawianie tutaj ścieżki bezwzględnej nie jestem dobrym pomysłem. Lepiej względną, bo wtedy niezależnie od struktury folderów na dysku, zawsze to będzie działać. Aczkolwiek ja jeszcze kiedyś sam tutaj pozmieniam jak należy.